app/src/main/kotlin/com/wire/android/notification/WireNotificationManager.kt

```diff
@@ -9,6 +9,7 @@ import com.wire.kalium.logic.CoreLogic
 import com.wire.kalium.logic.data.id.ConversationId
 import com.wire.kalium.logic.data.id.QualifiedID
 import com.wire.kalium.logic.data.notification.LocalNotificationConversation
+import com.wire.kalium.logic.data.sync.ConnectionPolicy
```

Comment:
is code-style checking okay about not used imports???

Reply:
Not in `Reloaded`, only in `Kalium`.

I'd say we can enable it here too... @yamilmedina what do you think? I can go ahead with this.